packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -622,6 +622,7 @@ export const Table = <
           dispatch({
             type: shiftRowSelectedAction,
             id: row.id,
+            ctrlPressed: event.ctrlKey,
```

Comment:
This doesn't work on macOS with <kbd>cmd</kbd> key. Usually you want to check for [`metaKey`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey) too.
```suggestion
            ctrlPressed: event.ctrlKey || event.metaKey,
```

And when I was testing this, I found that line 629 also needs to be changed:

```diff
-          (selectionMode === 'single' || !event.ctrlKey)
+          (selectionMode === 'single' || !(event.ctrlKey || event.metaKey))
```

Reply:
> This doesn't work on macOS with <kbd>cmd</kbd> key. Usually you want to check for [`metaKey`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey) too.

_(I'm guessing you meant the <kbd>ctrl</kbd> key)_. Thanks, I made that change.

Just curious, what is the Mac equivalent of a Ctrl on Windows/Linux? Like how to do ctrl clicks in Mac's file explorer for example? Using the <kbd>ctrl</kbd> key or the <kbd>command</kbd> key?

Reply:
> > This doesn't work on macOS with cmd key. Usually you want to check for [`metaKey`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey) too.
> 
> _(I'm guessing you meant the ctrl key)_. Thanks, I made that change.
> 
> Just curious, what is the Mac equivalent of a Ctrl on Windows/Linux? Like how to do ctrl clicks in Mac's file explorer for example? Using the ctrl key or the command key?

no i meant <kbd>cmd</kbd> (or <kbd>command</kbd> or <kbd>⌘</kbd>, all the same). that is the mac equivalent of <kbd>ctrl</kbd> on windows in most cases, including file explorer.

Reply:
**[DONE]** Ohh, makes sense, thanks. I made the change in the two places `event.ctrlPressed` was used ([Commits](https://github.com/iTwin/iTwinUI-react/pull/888/files/f37c20902961eb76118aa148f7a0d21f467688c2..3ca38c2439a04451eb9682099b198db22d97dcfa))